bundles/EcommerceFrameworkBundle/OrderManager/V7/OrderAgent.php

```diff
@@ -259,7 +259,7 @@ public function hasPayment()
      */
     public function getCurrency()
     {
-        return $this->environment->getDefaultCurrency();
+        return $this->getOrder()->getCurrency() !== null ? new Currency($this->order->getCurrency()) : $this->environment->getDefaultCurrency();
```

Comment:
Can we make the intention a bit clearer. Tenary operator seems to crowded for this one.
Does this really change the behaviour in the Back Office? I tried to set my Default Currency to `USD`. All my orders are in `EUR`, but this did not make any difference.  

```suggestion
        if($this->getOrder()->getCurrency() !== null) {
            return new Currency($this->getOrder()->getCurrency());
        } 
        return $this->environment->getDefaultCurrency();
```

Reply:
I just checked and we only load the default currency here
https://github.com/pimcore/pimcore/blob/10.5/bundles/EcommerceFrameworkBundle/Controller/AdminOrderController.php#L190

And later display it ike this
https://github.com/pimcore/pimcore/blob/10.5/bundles/EcommerceFrameworkBundle/Resources/views/admin_order/list.html.twig#L106

Reply:
for us it changes the behaviour:
Without patch (Order currency USD, default Currency CHF)
![image](https://user-images.githubusercontent.com/2370097/227169605-f278bbdc-8ab7-4639-aac8-9c7bc451b00c.png)

with patch applied:
![image](https://user-images.githubusercontent.com/2370097/227169860-56149247-4b1f-473a-bfaf-b28836fb66e2.png)

Reply:
@rliebi the committed changes do not resolve my second question. This actually does not change the back office.

Reply:
I checked again, and in the order itself it seems fine, but this does not change the listing.
imho it would only make sense to change the listing too, but then you have to divide the different currencies, because you cannot simply add them up, and we would need to provide a filter for the currencies too.

At the moment it is at least consistently wrong. With your change, you would get e.g. a list in CHF and if you click on the item you would see that it is USD. Feels wrong to me.

Reply:
what would you think about this solution?

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/2370097/227380670-91aa81d8-6f03-478d-807e-983dd82747ac.png">

styling could be better, sure, but I think this could work for list view.

Reply:
@rliebi Looks a lot better to be but it feels wrong to publish this in a bugfix version.
Would you please rebase your PR to 10.6

Reply:
> @rliebi Looks a lot better to be but it feels wrong to publish this in a bugfix version.
> Would you please rebase your PR to 10.6

I disagree with this one. Nothing changes if there is only one Currency displayed and it fixes if multiple currencies are displayed. No new feature was introduced but a bug was fixed.

I realigned the Totals to one line to make it look more clear:
I am open for suggestions :)
![image](https://user-images.githubusercontent.com/2370097/227622371-64f0be8b-9ee1-4d2a-9a85-b59532a7dc62.png)